lib/ipinfo/adapter.rb

```diff
@@ -33,10 +33,15 @@ def connection(adapter)
     end
 
     def default_headers
-        {
-          'User-Agent' => "ipinfo-ruby/#{::IpinfoIo::VERSION}",
+        headers = {
+          'User-Agent' => "IPinfoClient/Ruby/1.0",
```

Comment:
It seems like a duplication -- version number is stored in version.rb file, but your hardcoding it here. I think people will forget to increment it here.

Reply:
In this case, the version is actually intended to be the API version, not the library version.